en/publish/publish-wechatgame.md

```diff
@@ -7,21 +7,21 @@
+The running environment of the __WeChat Mini Game__ is an extension of the mini program environment. The basic idea is to provide the necessary web interface to the user, and pursue the same development experience as the web. The __WeChat Mini Game__ provides a wrapper around the WebGL interface based on the mini program environment, which greatly improves the rendering capabilities and performance. However, since these interfaces are encapsulated by the __WeChat__ team they are not equivalent to the browser environment. Regarding the game aspect, the current __WeChat Mini Game__ environment provides the rendering interface of __Canvas__ and __WebGL__. The two rendering modes of the __Cocos Creator__ engine can be run, but for performance reasons, we use __WebGL__ to render the game content by default. Developers are strongly recommended to do the same. Also, developers should not modify the default configuration!
 
 As an engine, in order to simplify the developer's workload as much as possible, the main tasks we have done for our users include:
 
-- The engine framework adapts to the WeChat Mini Game API, pure game logic level, users do not need any additional modifications
-- __Cocos Creator__ editor provides a fast packaging process, released directly as a WeChat game, and automatically evokes the developer tools of the game
-- Automatic loading of remote resources, cache resources and cache resource versioning
+- The engine framework adapts to the WeChat Mini Game API, pure game logic level, users do not need any additional modifications.
+- __Cocos Creator__ editor provides a fast packaging process, released directly as a __WeChat Mini Game__, and automatically evokes the developer tools of the mini game.
+- Automatic loading of remote resources, cache resources and cache resource versioning.
 
-In addition, the game submission, review and release process and small program of the __WeChat Mini Game__ platform are indistinguishable. They all need to comply with the requirements and standard processes of the __WeChat__ team. For details, please refer to the link at the end of the article.
+In addition, the game submission, review and release process of the __WeChat Mini Game__ platform is no different from the __WeChat Mini Program__ platform. They all need to comply with the requirements and standard processes of the __WeChat__ team. For details, please refer to the link at the end of the article.
 
 ## Using Cocos Creator to publish WeChat games
-We started with __Cocos Creator__ v1.8 and supported the __Cocos Creator__ game as a __WeChat__ game. Here are the detailed release steps:
+We started with __Cocos Creator__ v1.8 and support __WeChat Mini Game__  with just one-click to publish! Here are the detailed release steps:
 
 1. Download WeChat Developer Tools on [WeChat Public Platform](https://mp.weixin.qq.com/debug/wxagame/dev/devtools/download.html)
```

Comment:
The link has been replaced with the English version, please change it: https://developers.weixin.qq.com/miniprogram/en/dev/devtools/download.html